### PR TITLE
47 add testing for db

### PR DESCRIPTION
**Pull request by Fr3dMer**
- [x] Create test enviro for db 
- [x] Create tests for db

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -4,5 +4,5 @@ build-backend = "flit_core.buildapi"
 
 [project]
 name = "PannelAppDB"
-authors = [{name = "Frederick Mercer", email = "frederick.mercer@postgrad.manchester.ac.uk"}{name = "Abigail Haddon", email = "abigail.haddon@postgrad.manchester.ac.uk"}{name = "Amy Grimwood", email = "amy.grimwood@postgrad.manchester.ac.uk"}]
+authors = [{name = "Frederick Mercer", email = "frederick.mercer@postgrad.manchester.ac.uk"},{name = "Abigail Haddon", email = "abigail.haddon@postgrad.manchester.ac.uk"},{name = "Amy Grimwood", email = "amy.grimwood@postgrad.manchester.ac.uk"}]
```
Comment (AbiHaddon) [inline pyproject.toml:7]: line 85 characters

File: test/database_3_0_test.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
+"""***************************************************************************
+File          : database_3.0_test.py
+About         : Test db class is working as expected 
+Author        : Freddie Mercer
+Date modified : 2024-01-25
+***************************************************************************"""
```
Comment (AbiHaddon) [inline test/database_3_0_test.py:6]: 115 characters

File: test/database_3_0_test.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+
+def test_insert_patient_record(db_env):
+    # Test insert patient record func
+    
+    assert os.path.exists("tmp_test") == True
+    assert db_env.insert_patient_record('EX100', 
+                                        'PID10', 
+                                        'R111-1.1-1', 
+                                        datetime.date(2023, 12, 10)) == True
```
Comment (AbiHaddon) [inline test/database_3_0_test.py:40]: image
Again, pep8. Can be rectified if got time

**Review by AbiHaddon — APPROVED** [review]
Looks good happy to approve. Changes can be made after integration.

File: test/database_3_0_test.py (hunk 3 above)
Comment (Fr3dMer) [reply test/database_3_0_test.py:40]: Feature to be added for future releases

**Pull request merged**